package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "private": true,
   "engines": {
-    "node": ">=6 <=8"
+    "node": "<=8.10"
```

Comment:
Did you mean `>=8.10` ?

Reply:
heh, yeah >= 8.10 is actually better... but we don't use this config anyway (see: https://github.com/yarnpkg/yarn/issues/2800)